service/src/main/resources/sql/billing-account/budget-amount/update-consumed-amount-for-challangeid.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE project_challenge_budget
+    SET locked_amount = :lockAmount, 
```

Comment:
The name `lockAmount` does not match the binding parameter in DAO, which is `lockedAmount`